src/messages/formatters.ts

```diff
@@ -145,6 +146,33 @@ export function hyperlink(content: string, url: string | URL, title?: string) {
 	return title ? `[${content}](${url} "${title}")` : `[${content}](${url})`;
 }
 
+/**
+ * Formats the user ID into a user mention.
+ * @param userId The user ID to format.
+ * @returns The formatted user mention.
+ */
+export function userMention<C extends Snowflake>(userId: C): `<@!${C}>` {
```

Comment:
I think this should provide an option that allows us to choose if we want to use the `!` or not.

Reply:
`!` should always be added in `memberMention`, but never in `userMention`.

Reply:
> `!` should always be added in `memberMention` (missing here), but never in `userMention`.

Well the `!` only means that a member has a nickname, it is not very useful. The developer should at least have the choice (or you can name the method like `memberNicknameMention`, which is more accurate)

Reply:
What's the difference between adding the `!` and omitting it?